src/tpm2tss_engine.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Xaptum, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <openssl/ui.h>
+
+#ifndef ENFTUN_SSL_UI_H
+#define ENFTUN_SSL_UI_H
+UI_METHOD *
+enftun_ssl_ui_passthrough(void);
+
+int
+enftun_tpm2tss_ssl_ctx_set(const char *key_file, SSL_CTX* ctx);
```

Comment:
Since this is a "public" function, it should take the `enftun_tls` object. Also the `ssl_ctx_set` suffix isn't very descriptive or precise.

It's use is a parallel to `SSL_CTX_use_PrivateKey_file`, but could be a file or handle, so how about:

How about
```c
int
entun_tls_tpm_use_key(enftun_tls* tls, const char* key_file);
```

Reply:
Changed